csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -1012,7 +1012,7 @@ module Private {
       private predicate relevantSummaryElementGenerated(
         AccessPath inSpec, AccessPath outSpec, string kind
       ) {
-        summaryElement(this, inSpec, outSpec, kind, "generated") and
```

Comment:
Sorry I missed this in the original PR for introducing the *ai-generated* provenance.
I will make sure to re-write the code such that we don't need to change this in multiple places going forward (it will be a part of the next small epic for this area). I hope to get started soon in any case.